internal/pkg/template/template_functions.go

```diff
@@ -154,6 +154,36 @@ func generateSNSJSON(topics []*Topic) string {
 	return string(out)
 }
 
+// generateQueueURIJSON turns a list of Topic Subscription objects into a JSON string of their corresponding queues:
+// `{"EventsQueue": "${mainURL}", "svc-topicEventsQueue": "${svc-topicURL}"}`
+// This function must be called on an array of correctly constructed Topic objects.
+func generateQueueURIJSON(ts []*TopicSubscription) string {
+	if ts == nil {
+		return ""
+	}
+	urlMap := make(map[string]string)
+	urlMap["EventsQueue"] = "${mainURL}"
```

Comment:
We should probably keep it consistent with the other json injection such that the key is `snakeCaseCaseSnake` - sorry for not seeing this in my previous review!

Reply:
So the json string would be 
`{"eventsQueue": "${mainURL}", "svcTopicEventsQueue": "${svc-topicURL}"}`